apps/docs/components/Navigation/NavigationMenu/MenuIcons.tsx

```diff
@@ -684,7 +684,7 @@ export function IconUI({ width = 16, height = 16, className }: HomeMenuIcon) {
         height="14.72"
         rx="1.92"
         stroke="currentColor"
-        stroke-width="1.28"
+        strokeWidth="1.28"
```

Comment:
Unrelated fix. Bug introduced in https://github.com/supabase/supabase/pull/38827